database/migrations/2023_02_13_113535_create_orders_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained();
+            $table->bigInteger('price');
+            $table->enum('payment_status', ['successful', 'failed', 'pending']);
```

Comment:
no no, never, ever, do not use MySQL enum.
just do it as string. or tiny int ( based on your enum value )